firmware/src/deps/.gitmodules

```diff
@@ -1,6 +0,0 @@
-[submodule "avr-usi-spi"]
```

Comment:
I'm not sure why we had this second `.gitmodules` file. It seems unnecessary to me.